src/py/flwr/server/app.py

```diff
@@ -441,6 +457,29 @@ def run_server() -> None:
         driver_server.wait_for_termination(timeout=1)
 
 
+def _try_obtain_certificates(
+    args: argparse.Namespace,
+) -> Optional[Tuple[bytes, bytes, bytes]]:
+    # Obtain certificates
+    if args.insecure:
+        log(WARN, "The server will be started without HTTPS.")
```

Comment:
I would word this differently.

```suggestion
        log(WARN, "Option `--insecure` was found. Starting insecure HTTP server.")
```